rbi/stdlib/fileutils.rbi

```diff
@@ -99,6 +99,8 @@ module FileUtils
   OPT_TABLE = T.let(T.unsafe(nil), T::Hash[T.untyped, T.untyped])
   VERSION = T.let(T.unsafe(nil), String)
 
+  module_function
```

Comment:
It looks like in the ruby source, `module_function` is applied to individual methods: are you sure that every method with a sig that we have in the RBI should be marked `module_function`?

Reply:
`module_function` when called standalone does apply to all subsequent methods, just like `private` does in a class.
[apidoc documentation](https://apidock.com/ruby/Module/module_function)

Reply:
Yes, my question was: should we apply `module_function` to all methods, given that in the fileutils.rb source in the standard library, they apply it on an individual basis.

Reply:
I'm sorry I deleted last comment. Yes, I realized that was what you meant, and actually looking at fileutils.rb it seems that some methods are marked with `private_module_fuction`

Reply:
I had a quick glance and it doesn't seem like there are any methods in here which are marked with `private_module_fuction` in `fileutils.rb`.